src/sdk/model/company/support-ticket/__json__/support-ticket-create-request-json.ts

```diff
@@ -0,0 +1,23 @@
+export type ContactType = 'PHONE' |
```

Comment:
we're putting type definitions like these in their own files these days. filename for each should end in -type